mackerel-plugin-postgres/lib/postgres.go

```diff
@@ -239,6 +259,56 @@ func fetchDatabaseSize(db *sqlx.DB) (map[string]interface{}, error) {
 	}, nil
 }
 
+var versionRe = regexp.MustCompile("PostgreSQL (\\d+)\\.(\\d+)(\\.(\\d+))? ")
+
+type version struct {
+	first  uint
+	second uint
+	thrird uint
+}
```

Comment:
I thought major/minor would be better at a glance but, Okay, postgresql seems to change the semantics since version 10 due to the document.